skbeam/core/tests/test_correlation.py

```diff
@@ -88,13 +101,43 @@ def test_lazy_multi_tau():
     assert np.average(full_result.g2-1) < 0.01
 
     # run the correlation on the first half
-    gen_first_half = lazy_multi_tau(
+    gen_first_half = lazy_two_time(rois, img_stack[:stack_size//2], stack_size,
+                                   num_bufs=stack_size, num_levels=1)
```

Comment:
Can `num_frames` be longer than the length of the image stack?  Here `len(img_stack)` is only half of `stack_size`